tutorials/dagster-serverless-jobs/index.mdx

```diff
@@ -0,0 +1,190 @@
+---
+meta:
+  title: Managing Serverless Jobs using Dagster
+  description: This page shows how to create and manage Serverless Jobs using Dagster.
+content:
+  h1: Creating and deploying an Angular application on Serverless Containers
+  paragraph: This page shows how to create and manage Serverless Jobs using Dagster.
+tags: serverless jobs dagster python sdk orchestrator serverless-database
+categories:
+  - container-registry
+  - jobs
+  - object-storage
+  - serverless-sqldb
+dates:
+  validation: 2024-01-23
+  posted: 2024-01-23
+---
+
+Dagster is an orchestrator designed for developing and maintaining data assets, such as tables, data sets, machine learning models, and reports.
+
+You declare jobs that you want to run and the data assets that those jobs produce or update. Dagster then helps you run your functions at the right time and keep your assets up-to-date.
+
+In this tutorial, we will create a [Serverless Job](/serverless/jobs/quickstart) using Dagster to scrape the Hacker News webstite to retrieve the top articles' titles and URL.
+
+<Message type="requirement">
+  - You have an account and are logged into the [Scaleway console](https://console.scaleway.com)
+  - You have a [Container Registry namespace](/containers/container-registry/how-to/create-namespace/) in the Paris region and have [signed in to it](/containers/container-registry/how-to/connect-docker-cli/)
+  - You have created a [Serverless Database](/serverless/sql-databases/how-to/create-a-database/)
+  - You have installed [Docker](https://docs.docker.com/engine/install/) on your local machine to build and push the Docker image
+  - You have [configured your SSH key](/console/project/how-to/create-ssh-key)
+  - You have [created an Object Storage bucket](/storage/object/how-to/create-a-bucket/) in the Paris region
+</Message>
+
+## Creating the Dagster project
+
+1. Create a new folder and access it:
```

Comment:
Not sure if this step is necessary since `dagster project scaffold` already create all of its resources in a new directory